packages/invoices/src/OutboundShipment/DetailView/tabs/GeneralTab.tsx

```diff
@@ -60,14 +58,14 @@ const Expand: FC<{
   if (!rowData?.canExpand) return <></>;
 
   return (
-    <Box p={1} style={{ padding: '0 50px 0 200px' }}>
+    <Box p={1} style={{ padding: '0 100px 0 100px' }}>
```

Comment:
I didn't like it without some padding, the full width version looks odd to me. I also didn't like having the left hand edge more left than the first column in the summary, but agree that it isn't great. thought it was slightly better when centred?

Reply:
thanks. yep - have fixed by removing the `100%` width on the grid, which was constraining it; that was in there to get the switch to sit on the RH side, so I've moved that over to the left, which I don't like as much, though it does fix the problem of the switch sitting in the middle when the table stretches and you scroll over a bit.